docs/calendar.md

```diff
@@ -124,7 +124,7 @@ Example usage:
 ```javascript
 cal = new Temporal.Calendar('iso8601');
 cal = new Temporal.Calendar('gregory');
-/*⚠️*/ cal = new Temporal.Calendar('discordian'); // not a built-in calendar, throws
```

Comment:
I can't remember exactly but I think someone had asked for the :warning: sign on intentionally bad examples so that they're harder to unwittingly copy and paste.

Reply:
ah, many places just had `=> throws` and only a few had the :warning: so I went with the former - do you want me to put them back in or change all throw examples to have the emoji?

The third convention was /* WRONG */, which I also eliminated.

Reply:
Making it consistently either `/*⚠️*/` or `/* WRONG */` is fine with me, in addition to `=> throws`. I did get feedback specifically that `=> throws` was not obvious enough.